user_docs/en/changes.t2t

```diff
@@ -11,6 +11,13 @@ What's New in NVDA
 == Changes ==
 - NSIS has been updated to version 3.08. (#9134)
 - Added new role for "busy indicator" controls. (#10644)
+- NVDA now announces when an NVDA action cannot be performed. (#13500)
+  - This includes when:
+    - using the NVDA Windows Store version
+    - in a secure context
+    - waiting for a response to a modal dialog

```

Comment:
They could be written as additive clauses.

```suggestion
  - This includes when:
    - using the NVDA Windows Store version;
    - in a secure context; and
    - waiting for a response to a modal dialog.

```

Reply:
The additive clause style is inconsistent with the rest of the doc and isn't typical of a technical documentation.